apps/web/src/components/Markdown/courses-markdown-body.tsx

```diff
@@ -51,7 +51,7 @@ const CoursesMarkdownBody = ({
         ),
         p: ({ children }) => (
           <p className="text-blue-1000 text-base tracking-wide md:text-justify">
-            {children}
+            <code className="whitespace-pre-wrap break-words">{children}</code>
```

Comment:
Not good as it would change the semantic

Use https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state instead